src/main/java/com/github/thorbenkuck/netcom2/network/shared/clients/DefaultSendingService.java

```diff
@@ -218,6 +219,7 @@ public Awaiting started() {
 	 */
 	@Override
 	public void setConnectionIDSupplier(Supplier<String> supplier) {
```

Comment:
Should this be `final` too?

Reply:
Should be, yes. Was not part of this issue tho. Should be done seperatly.